src/test/java/org/example/CalendarManagement/calendarservice/implementation/MeetingRoomServiceTest.java

```diff
@@ -0,0 +1,107 @@
+package org.example.CalendarManagement.calendarservice.implementation;
+
+import org.example.CalendarManagement.calendarpersistence.model.MeetingRoom;
+import org.example.CalendarManagement.calendarpersistence.repository.MeetingRoomRepository;
+import org.example.CalendarManagement.thriftclients.interfaces.MeetingServiceClient;
+import org.example.CalendarThriftConfiguration.Date;
+import org.example.CalendarThriftConfiguration.FindFreeMeetingRoomDataRequest;
+import org.example.CalendarThriftConfiguration.MeetingRoomAvailableDataRequest;
+import org.example.CalendarThriftConfiguration.Time;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MockitoExtension.class)
+class MeetingRoomServiceTest {
+    @Mock
+    MeetingServiceClient meetingServiceClient;
+    @Mock
+    MeetingRoomRepository meetingRoomRepository;
+    @InjectMocks
+    MeetingRoomService meetingRoomService;
+
+    @Test
+    public void meetingRoomServiceTest_findFreeMeetingRoom_RoomsClosed(){
```

Comment:
can these names be like findFreeMeetingRoomTest_roomsClosed ? because the tests are already in meetingRoomService ?

Reply:
yes they can be